partner_priority/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_partner_priority_user,partner.priority.user,model_partner_priority,base.group_user,1,1,1,1,

```

Comment:
This gives any employee the possibility to edit the priorities, you might consider two separate entries, one for group base.group_partner_manager with all rights, and one for base.group_user with only read rights: 1,0.0.0.

Reply:
Good idea, i've added that manager permission.